app/config.js

```diff
@@ -15,9 +15,13 @@
 
 /* global __TEST__ */
 
+if (__TEST__ && (window === undefined || window.config === undefined)) {
```

Comment:
@jebeck this was the update that allowed the tests to work. `personutils.js` now has a reference to config and that was killing the tests.